browser/net/brave_network_delegate_base.h

```diff
@@ -20,6 +28,17 @@ namespace net {
 class URLRequest;
 }
 
+namespace brave {
```

Comment:
nit: using namespace here looks inconsistent with other code in this file. Don't really know if it is better to not use it here.